lib/templates/middleAssignment/data/boxes.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:hive_flutter/adapters.dart';
+
+class Boxes {
```

Comment:
If tried to exit and then add the session

<img width="902" alt="Screenshot 2024-06-24 at 11 34 42" src="https://github.com/timur-harin/Sum24FlutterCourse/assets/64702874/9019892c-9833-47f8-9d90-ef713c63a2bf">